sync/requirements.txt

```diff
@@ -1,5 +1,5 @@
 wget==3.2
-PyYAML==5.1.2
+ruamel.yaml==0.16.12
```

Comment:
This change is so that we can preserve comments in the YAML config when updating them via a python script.